From 1ec6dfa831c3e326abff7973c949ad6f1ce803b4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 8 May 2017 20:38:44 +0000
Subject: [PATCH/RFC 2/6] Makefile & compat/pcre2: add dependency on
 pcre2_convert.c
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add a dependency on the experimental pcre2_convert.c. This only exists
in svn trunk of pcre2 currently, and allows for converting POSIX
basic/extended & glob patterns to patterns accepted by PCRE[1][2].

1. https://bugs.exim.org/show_bug.cgi?id=2106
2. https://bugs.exim.org/show_bug.cgi?id=2107

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile                  | 1 +
 compat/pcre2/get-pcre2.sh | 1 +
 2 files changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index b18867196e..e437fa011c 100644
--- a/Makefile
+++ b/Makefile
@@ -1541,6 +1541,7 @@ endif
 		compat/pcre2/src/pcre2_compile.o \
 		compat/pcre2/src/pcre2_config.o \
 		compat/pcre2/src/pcre2_context.o \
+		compat/pcre2/src/pcre2_convert.o \
 		compat/pcre2/src/pcre2_error.o \
 		compat/pcre2/src/pcre2_find_bracket.o \
 		compat/pcre2/src/pcre2_jit_compile.o \
diff --git a/compat/pcre2/get-pcre2.sh b/compat/pcre2/get-pcre2.sh
index f1796cb518..7679fba8e4 100755
--- a/compat/pcre2/get-pcre2.sh
+++ b/compat/pcre2/get-pcre2.sh
@@ -26,6 +26,7 @@ for srcfile in \
 	pcre2_compile.c \
 	pcre2_config.c \
 	pcre2_context.c \
+	pcre2_convert.c \
 	pcre2_error.c \
 	pcre2_find_bracket.c \
 	pcre2_jit_compile.c \
-- 
2.11.0

